doozer/doozerlib/cli/images_streams.py

```diff
@@ -831,6 +832,20 @@ def search_issues(query):
             new_issues[distgit_key] = issue
             print(f'A JIRA issue has been opened for {pr.html_url}: {issue.key}')
             connect_issue_with_pr(pr, issue.key)
+            try:
+                # update created issue by missing custom fields
+                # Retrieve the issue object
+                issue = jira_client.issue(issue_key)
```

Comment:
We already have an `issue` before. Can we just use that object rather than create a new one?

Reply:
resolved with the latest update